addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -4,6 +4,23 @@ import options from 'web_editor.snippets.options';
 
 options.registry.WebsiteEvent.include({
 
+    /**
+     * @override
+     */
+    async start() {
+        const res = await this._super(...arguments);
+        const rpcData = await this._rpc({
```

Comment:
"rpcData" -> "eventData"

Reply:
Not really important, you can keep as is for consistency with the other file indeed, I let you choose :slightly_smiling_face: